### PR TITLE
MAINT: Avoid bad VTK [skip actions] [skip circle]

### PR DESCRIPTION
**Pull request by larsoner**

### CODE REVIEW

File: server_environment.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ channels:
 - conda-forge/label/vtk_dev
 - conda-forge
 dependencies:
-- python>=3.7
+- python ==3.8
```
Comment (hoechenberger) [inline server_environment.yml:6]: why the pinning?

Comment (larsoner) [reply server_environment.yml:6]: segfaults on 3.9 for unknown reasons

Comment (hoechenberger) [reply server_environment.yml:6]: Ouch. Has this been reported upstream?

Comment (larsoner) [reply server_environment.yml:6]: No, I haven't tried to replicate locally or anything. Feel free if you have time

Comment (hoechenberger) [reply server_environment.yml:6]: Also, do we actually need the VTK dev channel any longer?

Comment (larsoner) [reply server_environment.yml:6]: Not sure, maybe that's the problem with the bump to 3.9

Comment (hoechenberger) [reply server_environment.yml:6]: Will have a look.

**Pull request merged**